crates/bevy_render/src/view/mod.rs

```diff
@@ -87,6 +87,7 @@ impl Default for Msaa {
 pub struct ExtractedView {
     pub projection: Mat4,
     pub transform: GlobalTransform,
+    pub view_projection: Option<Mat4>,
```

Comment:
would it be possible to remove the old projection field and just do the projection * inverse calculation using `view_projection` wherever `projection` is currently used? then you have the fine control you need without duplicating the data?

if not then can we add some comments here. it seems like a source of potential future confusion to have a `projection` and a `view_projection` field with no descriptions.

Reply:
after discussing with robswain on discord, it's fine to keep both, pls disregard this.

Reply:
Keeping field, but I added a comment explaining why it's optional and when you'd want to use it.